.semaphore/semaphore.yml

```diff
@@ -83,7 +83,10 @@ blocks:
           - sudo pip3 install meson
           - echo "Checking out..."
           - checkout
-          - git submodule update --init --recursive
+          # Clean up the 3rdparty folder (aubio is installed from system pkgs) and pull ced from its repository
+          - rm -rf 3rdparty
+          - git clone https://github.com/google/compact_enc_det.git 3rdparty/ced
+          - cd 3rdparty/ced & git checkout -q 37529e6 & cd ../..
```

Comment:
Instead of doing this from within the CI i'd expect this to be done from the meson script just like we do with cmake in #595 
The meson variant for this would be 'wraps': https://github.com/mesonbuild/meson/blob/master/docs/markdown/Wrap-dependency-system-manual.md

Reply:
I agree this would be better.

As this PR is an improvement over the current situation, should it be merged meanwhile (after CI is re-established?)

Reply:
It can be merged. But currently the pipeline of the meson build fails:
```
echo "Setting up build..." Passed in 00:01
meson setup build.clang -Dusewebcam=true -Dusemididrum=true -Dusewebserver=true Failed in 00:00
Traceback (most recent call last): 00:00
  File "/usr/local/lib/python3.6/dist-packages/mesonbuild/mesonmain.py", line 134, in run 00:00
    return options.run_func(options) 00:00
  File "/usr/local/lib/python3.6/dist-packages/mesonbuild/msetup.py", line 280, in run 00:00
    app = MesonApp(options) 00:00
  File "/usr/local/lib/python3.6/dist-packages/mesonbuild/msetup.py", line 76, in __init__ 00:00
    options.wipe) 00:00
  File "/usr/local/lib/python3.6/dist-packages/mesonbuild/msetup.py", line 157, in validate_dirs 00:00
    (src_dir, build_dir) = self.validate_core_dirs(dir1, dir2) 00:00
  File "/usr/local/lib/python3.6/dist-packages/mesonbuild/msetup.py", line 128, in validate_core_dirs 00:00
    os.makedirs(ndir1) 00:00
  File "/usr/lib/python3.6/os.py", line 220, in makedirs 00:00
    mkdir(name, mode) 00:00
PermissionError: [Errno 13] Permission denied: '/home/build.clang'
```

Reply:
Strange.  Nothing changed ? Maybe re-run the pipeline ?